transformations/add_hashtag/transformation.py

```diff
@@ -0,0 +1,118 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+import random
+from spacy import load
+
+be_class_verb = ["is", "am", "are", "was", "were", "will", "shall"]
+subj_obj_list = ["i", "you","we", "they", "he", "she"]
+nlp=load('en_core_web_sm')
```

Comment:
This should be placed inside the init function - else it will loaded even when we don't load this generator.